applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_multi_contact_test_materials.json

```diff
@@ -0,0 +1,45 @@
+{
+    "properties" : [{
+        "model_part_name" : "Parts_Lower",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw"
```

Comment:
`LinearElastic3DLaw` is enough, the rest is ignored anyway...

Reply:
Sure, @jginternational can we change this?